scripts/auto-rebase/assets.yaml

```diff
@@ -22,10 +22,13 @@ assets:
       - file: update-node-resolver.sh
 
   - dir: components/openshift-router/
+    no_clean: true
     src: cluster-ingress-operator/pkg/manifests/assets/router/
     files:
       - file: cluster-role-binding.yaml
       - file: cluster-role.yaml
+      - file: cluster-role-system-router.yaml
+        ignore: "it's manually copied from OAS bootstrap policy. Not available in any repo rebase is using"
```

Comment:
Could we use `git_restore` and not add `no_clean`?

Reply:
Seems to work according to https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_microshift/2512/pull-ci-openshift-microshift-main-test-rebase/1717915044823437312/artifacts/test-rebase/openshift-microshift-rebase/build-log.txt